cvpy/tests/test_imagetable.py

```diff
@@ -266,6 +266,139 @@ def test_imagetable_from_table_custom_col_names(self):
 
         assert image_table.table.name == cas_table.name
         assert type(image_table) == NaturalImageTable
+    
+    # Load client images and server images using load_client_images and loadImages and compare results
+    def test_imagetable_load_client_images(self):
+        # For determining if we are working from a local machine or the Jenkins server
+        # and comparing images for verification
+        from PIL import ImageChops, ImageStat
+        from os.path import exists
+
+        # Path to the directory the function will load images from
+        path = "images/"
+
+        # If path exists, we are on the Jenkins server
+        if exists(path):
+            # Images are determined to be similar if the average pixel value of all channels
+            # are less than three percent of their maximum value.
+            validThreshold = 255 * 0.03
+
+            # Test loading images fron both client and server. One will have a valid path and
+            # return value to test the type, the other will try to load from a nonexistent directory
+            # and the last will load images from the server
+            natural = ImageTable.load_client_images(
+                output_table_parms={'name': 'imgsClientDirectory', 'caslib': 'CASUSER(user)'},
+                path=path,
+                connection=self.s,
+                subdirs=False
+            )
+            ImageTable.load_client_images(
+                output_table_parms={'name': 'imgsClientEmpty', 'caslib': 'CASUSER(user)'},
+                path='path/does/not/exist',
+                connection=self.s,
+                subdirs=False
+            )
+            self.s.CASTable('imgsServer', replace=True)
+            self.s.image.loadimages(
+                path="images/",
+                caslib="dlib",
+                decode=True,
+                casout={'name' : 'imgsServer'}
+            )
+            
+            # Perform the same process on both client and server images
+            self.s.CASTable('imgsOutClient', replace=True)
+            self.s.CASTable('imgsOutServer', replace=True)
+            self.s.image.processimages(
+                table={'name': 'imgsClientDirectory'},
+                casout={'name': 'imgsOutClient', 'replace': True},
+                steps=[{'step':{'stepType':'RESIZE', 'type': 'BASIC', 'width': 200, 'height': 200}}]
+            )
+            self.s.image.processimages(
+                table={'name': 'imgsServer'},
+                casout={'name': 'imgsOutServer', 'replace': True},
+                steps=[{'step':{'stepType':'RESIZE', 'type': 'BASIC', 'width': 200, 'height': 200}}]
+            )
+
+            # Compare results to see if they are affected in the same way
```

Comment:
Instead of python PIL functions, you can use compareimages action for image comparison.

Reply:
I've updated all the tests to use compareimages instead of PIL functions.